templates/taxi/driver_form.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'base.html' %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <h1>{{ object|yesno:'Update,Create' }} Driver</h1>

```

Comment:
The use of the `yesno` filter to toggle between 'Update' and 'Create' is clever. Ensure that the `object` variable is correctly set in the view context to avoid any unexpected behavior.